tests/flytekit/unit/common_tests/test_translator.py

```diff
@@ -92,6 +92,7 @@ def t1(a: int) -> (int, str):
         output_data_dir="/tmp",
         command=["cat"],
         arguments=["/tmp/a"],
+        requests=Resources(mem="400Mi", cpu="1"),
```

Comment:
lets add an example without the resources specified. (I know you have the defaulting behavior, but just a gate)